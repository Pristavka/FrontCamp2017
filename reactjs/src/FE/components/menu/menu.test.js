import React from 'react';
import Menu from './menu';

it('Should render a Menu', () => {
  const wrapper = shallow(<Menu />);
  expect(wrapper).toMatchSnapshot();
});

