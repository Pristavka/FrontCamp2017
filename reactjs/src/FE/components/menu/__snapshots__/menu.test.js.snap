// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Should render a Menu 1`] = `
<React.Fragment>
  <div
    className="slogan"
  >
    Blogs Portal
  </div>
  <Link
    replace={false}
    to="/"
  >
    Show all posts
  </Link>
  <Link
    replace={false}
    to="/addpost"
  >
    Add post
  </Link>
</React.Fragment>
`;
